packages/calculator/src/main.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Calculator from '@pie-framework/material-ui-calculator';
+import './style.css';
+
+class Main extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { showCalcByState: false };
+    }
+
+    showCalculatorHandler() {
+        this.setState({ showCalcByState: true });
+    }
+
+    hidecalculatorHandler() {
+        this.setState({ showCalcByState: false });
+    }
+
+    dragHandler = () => {
+        this.dragElement(document.getElementById("calculator-div"));
+    }
+
+    dragElement = (elmnt) => {
```

Comment:
Did you look at any drag libraries for react? Might be no harm. These are the 2 big ones: 
* https://github.com/mzabriskie/react-draggable
* https://github.com/react-dnd/react-dnd

I think I prefer to use `react-draggable` but have a look.